src/components/filters/filters.js

```diff
@@ -0,0 +1,35 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.filters', []);
+})(angular);
+
+(function (angular) {
+    'use strict';
+    function formatTime() {
+        return function(time) {
```

Comment:
for extended functionality u can add optional expression as parameter .
Lets discuss